website/cue/reference/releases/0.19.2.cue

```diff
@@ -0,0 +1,67 @@
+package metadata
+
+releases: "0.19.2": {
+	date:     "2022-02-08"
+	codename: ""
+
+	whats_next: []
+
+	description: """
+		The Vector team is pleased to announce version 0.19.2!
+
+		This patch release contains a few bug fixes for regressions in 0.19.0.
```

Comment:
Should this be `0.19.1`?

Reply:
Yeah, exactly, this release just fixes regressions in 0.19.0.